workbench/src/renderer/userguideURL.js

```diff
@@ -0,0 +1 @@
+export default 'https://storage.googleapis.com/releases.naturalcapitalproject.org/invest-userguide/latest';
```

Comment:
I like this pattern of having a separate file to export a single hard-coded value. Curious if this is an "official" pattern that you've heard of, or your invention?

Reply:
I think it's conventional. At an abstract level it's not really different than exporting a function or anything else that needs to be shared across more than one module.

But if we accumulated a lot of these shared constants, I think it's personal preference whether we have them each in their own module or have many exported from the same module.